src/bin/tikv-server.rs

```diff
@@ -812,6 +816,16 @@ fn get_store_labels(matches: &ArgMatches, config: &toml::Value) -> HashMap<Strin
         .unwrap_or_else(|err| exit_with_err(format!("{:?}", err)))
 }
 
+fn check_compression_available(compression: DBCompressionType) -> bool {
+    supported_compression().contains(&compression)
+}
+
+fn get_fastest_supported_compression_type() -> DBCompressionType {
+    *COMPRESSION_PRIORITY.into_iter()
```

Comment:
It's better to get all supported compression type and find the desire one. Hence a vec won't be created every time.